.circleci/config.yml

```diff
@@ -357,3 +387,16 @@ workflows:
               branches:
                 ignore: /.*/
           <<: *requires_all
+      - waiter:
+          filters:
+            branches:
+              only: master
+      - slack-notifier:
+          matrix:
+            parameters:
+              pythonversion: [3.10.2]
```

Comment:
hm, can we use some parameter here? e.g. set a param called `pythonversion_latest`, so we can change it when necessary instead of changing here